cmd/manager/kodata/knative-eventing/0.13.3.yaml

```diff
@@ -324,29 +324,6 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
-metadata:
```

Comment:
@aliok  why was that there anyways ?

Reply:
this has 0 replicas. So, it is not really used.